cloud/filestore/libs/vfs_fuse/fs_impl.cpp

```diff
@@ -263,7 +263,17 @@ void TFileSystem::ProcessHandleOpsQueue()
         return;
     }
 
-    const auto& entry = HandleOpsQueue->Front();
+    const auto optionalEntry = HandleOpsQueue->Front();
+    if (!optionalEntry.has_value()) {
+        ReportHandleOpsQueueProcessError("Failed to get TQueueEntry from queue");
+        with_lock(HandleOpsQueueLock) {
```

Comment:
you already creating guard which will lock the mutex in the beginning of this function
trying to lock the mutex again will lead to deadlock